### ISSUE TITLE
When i train Rain_Heavy

### ISSUE THREAD

**Issue by hellogry** [swz30/MPRNet#66]
the dataset has1800 training data, 200 test data,
Why the loss is so high, I try to change the learning rate to 1.5e-4. but it is no helpful

![image](https://user-images.githubusercontent.com/12219867/126451485-cd4ad93d-b317-4ffc-ae4b-32b7ca178f29.png)

**Comment by swz30** [comment]
Hi @hellogry 

Try this: https://github.com/swz30/MPRNet/issues/51#issuecomment-843802843

**Comment by hellogry** [comment]
I've used it and it doesn't work

**Comment by swz30** [comment]
Can you share the training log for few epochs? It will be fine as long as the loss is gradually decreasing (and not exploding).

**Comment by hellogry** [comment]
All losses remained almost at around 1442. The minimum is 1441. How do I display the training log.

**Comment by adityac8** [comment]
Hi @hellogry 

To save the logs, you can use 
`python train.py | tee deraining_logs.txt`

Thanks

**Comment by FanChiMao** [comment]
Isn't this normal? I'm not sure about that but I think that it's because the loss is epoch loss, the sum of every iteration loss. Divide by the training images, the iteration loss is about 0.8. It's acceptable and correct I think.
